scripts/scil_compute_kurtosis_metrics.py

```diff
@@ -0,0 +1,322 @@
+
+
+def identify_shells(bvals, threshold=40.0):
+    centroids = _guess_bvals_centroids(bvals, threshold)
+
+    bvals_for_diffs = np.tile(bvals.reshape(bvals.shape[0], 1),
+                              (1, centroids.shape[0]))
+
+    shell_indices = np.argmin(np.abs(bvals_for_diffs - centroids), axis=1)
+
+    return centroids, shell_indices
+
+
+def _build_args_parser():
+    p = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.RawTextHelpFormatter)
+    p.add_argument('input',
+                   help='Path of the input multi-shell DWI dataset.')
+    p.add_argument('bvals',
+                   help='Path of the bvals file, in FSL format.')
+    p.add_argument('bvecs',
+                   help='Path of the bvecs file, in FSL format.')
+
+    p.add_argument('--mask',
+                   help='Path to a binary mask.' +
+                   '\nOnly data inside the mask will be used '
+                   'for computations and reconstruction. ' +
+                   '\n(Default: None)')
+
+    p.add_argument('--tolerance', '-t',
+                   metavar='INT', type=int, default=20,
+                   help='The tolerated distance between the b-values to '
+                   'extract\nand the actual b-values [Default: %(default)s].')
+    p.add_argument('--min_k', dest='min_k', type=float, default='0',
+                   help='Minium kurtosis value in the output maps ' +
+                   '\n(ak, mk, rk). In theory, -3/7 is the min kurtosis ' +
+                   '\n limit for regions that consist of water confined ' +
+                   '\nto spherical pores (see DIPY example and ' +
+                   '\n documentation) [Default: %(default)s].')
+    p.add_argument('--max_k', dest='max_k', type=float, default='3',
+                   help='Maximum kurtosis value in the output maps ' +
+                   '\n(ak, mk, rk). In theory, 10 is the max kurtosis' +
+                   '\nlimit for regions that consist of water confined' +
+                   '\nto spherical pores (see DIPY example and ' +
+                   '\ndocumentation) [Default: %(default)s].')
+    p.add_argument('--smooth', dest='smooth', type=float, default='2.5',
+                   help='Smooth input DWI with a 3D Gaussian filter with ' +
+                   '\nfull-width-half-max (fwhm). Kurtosis fitting is ' +
+                   '\nsensitive and outliers occur easily. According to' +
+                   '\ntests on HCP, CB_Brain, Penthera3T, this smoothing' +
+                   '\nis thus turned ON by default with fwhm=2.5. ' +
+                   '\n[Default: %(default)s].')
+    p.add_argument('--not_all', action='store_true', dest='not_all',
+                   help='If set, will only save the metrics explicitly ' +
+                   '\nspecified using the other metrics flags. ' +
+                   '\n[Default: not set].')
+
+    g = p.add_argument_group(title='Metrics files flags')
```

Comment:
For the next options, remove dest option if `dest` is the same name than the option